HOK.Utilities/HOK.ColorBasedIssueFinder/ColorBasedIssueFinder.addin

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RevitAddIns>
+  <AddIn Type="Command">
+    <Text>Command HOK.ColorBasedIssueFinder</Text>
+    <Description>Some description for HOK.ColorBasedIssueFinder</Description>
+    <Assembly>HOK.ColorBasedIssueFinder.dll</Assembly>
+    <FullClassName>HOK.ColorBasedIssueFinder.Command</FullClassName>
+    <ClientId>999da13c-460e-423e-97f7-407bca86231c</ClientId>
+    <VendorId>com.typepad.thebuildingcoder</VendorId>
```

Comment:
Let's update VendorId to be your name and and VendorDescription to be `HOK DesignTech AppDev`